DylibBinder/StringExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DylibBinder {
+	internal static class StringExtensions {
+		internal static bool IsPublic (this string s)
+		{
+			if (s.Contains ("_"))
+				return false;
+			return true;
+		}
+
+		internal static string IsNull (this string s)
+		{
+			if (s == null)
+				return "True";
+			return "False";
+		}
```

Comment:
TJ, we don't need this at all, first is a one liner using one of the ?? operators, second we probably are doing something strange if we need this.